src/Symfony/Bundle/FrameworkBundle/Controller/ErrorController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Controller;
+
+use Symfony\Component\ErrorRenderer\ErrorRenderer;
+use Symfony\Component\ErrorRenderer\Exception\ErrorRendererNotFoundException;
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Renders error or exception pages for a given FlattenException.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class ErrorController
```

Comment:
As this only depends on `ErrorRenderer`, what about moving it to the `ErrorRenderer` component?

Reply:
I merged both controllers into one and moved it to the HttpKernel component, wdyt?

Reply:
Works for me